middleware/passport.js

```diff
@@ -42,24 +38,22 @@ passport.use(
     new JwtStrategy(
         jwtOptions, 
         async function (payload, done) {
-            let user;
-
             try{
-                user = await User.findById(payload.id);
+                const user = await User.findById(payload.id);
                 if (user) {
-                    done(null, user);
+                   return done(null, user);
                 } else {
-                    done(null, false);
+                   return done(null, false);
                 }
             } catch (error){
-                done(error);
+                return done(error);
             }
         }
     )
 );
 
 passport.serializeUser((user, done) => {
-    done(null, user.id);
+   return done(null, user.id);
```

Comment:
read this thread
https://stackoverflow.com/questions/27637609/understanding-passport-serialize-deserialize